paasta_tools/contrib/get_containers_and_ips.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+import argparse
+import os
+import socket
+
+from paasta_tools.utils import get_docker_client
+
+HAPROXY_STATS_SOCKET = '/var/run/synapse/haproxy.sock'
+
+
+def send_to_haproxy(command):
+    s = socket.socket(socket.AF_UNIX)
+    # 1 seconds should be more than enough of a timeout since HAProxy is local
+    s.settimeout(1)
+    s.connect(HAPROXY_STATS_SOCKET)
+    print("Sending {} to HAProxy stats socket".format(command))
```

Comment:
Yeah, I thought they could be useful and shouldn't matter much since it's just being run via cron, but that's a good point, they shouldn't really be here still.